support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -311,9 +333,22 @@ export function ThreeTierLanding(): JSX.Element {
 			<Container
 				sideBorders
 				borderColor="rgba(170, 170, 180, 0.5)"
-				cssOverrides={oneTimeContainer}
+				cssOverrides={oneTimeContainer(countryGroupId === UnitedStates)}
```

Comment:
Never know when to `dynamic` versus `static` with these css styles.  Would this be the `withShortPaddingBottom` variable is used >1 so dynamic is appropriate in this case?